### DISCUSSION TITLE
Why only  create `trimesh` static body?

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4763] [Editor]
![изображение](https://user-images.githubusercontent.com/16458555/175977003-8b9094fa-7314-42fa-afc8-e39e4beb8ef8.png)

Can it be for other collisions options too
like 
create simplified collision static body

**Comment by Calinou** [comment]
For static body collisions, trimesh is usually the right choice. If you don't need the complexity of the trimesh shape, manually add a StaticBody and a primitive collision shape instead.

We should be wary about adding too many options to this Mesh menu, as it has quite a lot of options for a context menu already.

**Reply by Delpire** [reply]
One option I could see being useful, is generating a box collision shape based on the AABB. When I participate in game jams and want things to run smoothly in the browser, I usually try to keep collision shapes as simple as possible. Often times this results in a lot of box collision shapes I have to manually create.

But I guess it could be a slippery slope as there are situations I want a cylinder (trees for example). Regardless I'd guess the 90% use case is generating box collision shapes for floors and walls, and it would be cool if it was just one button click away.

**Reply by Calinou** [reply]
This can be done on import already with the Advanced Import Options dialog. You can create primitive colliders for individual meshes in the scene and manually resize them (as using the culling AABB for collision isn't always suited).

**Reply by Zireael07** [reply]
@Calinou: Not all meshes are imported though - many of mine are either the built-in shapes like the cube scaled or created from SurfaceTool via code.

**Reply by Nodragem** [reply]
The real power of this menu is that we can select 100 meshes and click on `Create Trimesh Static Body`, and it applies it to the 100 meshes. 

Such Batch Processing is very practical when working with modular asset kits; but the Trimesh option is not necessarily what the user wants. We could have it replaced with `Batch Create Static Body` and open a menu which ask if we want AABB box, Trimesh or even other options.
